src/gui/advancedsettings.cpp

```diff
@@ -307,16 +304,19 @@ void AdvancedSettings::loadAdvancedSettings()
         .arg("https://github.com/qbittorrent/qBittorrent/wiki/Explanation-of-Options-in-qBittorrent#Advanced", tr("Open documentation")));
     labelQbtLink.setOpenExternalLinks(true);
     addRow(QBITTORRENT_HEADER, QString("<b>%1</b>").arg(tr("qBittorrent Section")), &labelQbtLink);
+    static_cast<QLabel *>(cellWidget(QBITTORRENT_HEADER, PROPERTY))->setAlignment(Qt::AlignCenter | Qt::AlignVCenter);
 
     labelLibtorrentLink.setText(QString("<a href=\"%1\">%2</a>").arg("https://www.libtorrent.org/reference.html", tr("Open documentation")));
     labelLibtorrentLink.setOpenExternalLinks(true);
     addRow(LIBTORRENT_HEADER, QString("<b>%1</b>").arg(tr("libtorrent Section")), &labelLibtorrentLink);
+    static_cast<QLabel *>(cellWidget(LIBTORRENT_HEADER, PROPERTY))->setAlignment(Qt::AlignCenter | Qt::AlignVCenter);
 
     // Async IO threads
     spinBoxAsyncIOThreads.setMinimum(1);
     spinBoxAsyncIOThreads.setMaximum(1024);
     spinBoxAsyncIOThreads.setValue(session->asyncIOThreads());
-    addRow(ASYNC_IO_THREADS, tr("Asynchronous I/O threads"), &spinBoxAsyncIOThreads);
+    addRow(ASYNC_IO_THREADS, (tr("Asynchronous I/O threads")
+            + " <a href=\"https://www.libtorrent.org/reference-Settings.html#aio_threads\">(?)</a>"), &spinBoxAsyncIOThreads);
```

Comment:
I would add some helper to enclose links with appropriate tags:
```c++
addRow(ASYNC_IO_THREADS, (tr("Asynchronous I/O threads") + makeLink("https://www.libtorrent.org/reference-Settings.html#aio_threads"))
        , &spinBoxAsyncIOThreads);
```
I'm not insisting that you do this right now.

Reply:
I'll do it here.

Reply:
Added another commit, a little modified from your suggestion. I'm open to any further changes and if you want me to merge it with first commit.